docs/gds-supported-platform/resizing-persistent-volumes.md

```diff
@@ -0,0 +1,96 @@
+# Resizing persistent volumes
+
+The volumes you created in the beginning may not be big enough for
+your ongoing needs.
+
+As an example, we're going to double the space on the Concourse Worker
+persistent volumes - from 64Gi to 128Gi. Replace
```

Comment:
If gigabytes, use GB

Reply:
Can we compromise on [GiB](https://www.iec.ch/si/binary.htm)?

Reply:
Ok